docs/DOMAIN/DOM-CLASS-003_ECONOMIC_POLICY.md

```
@@ -291,43 +281,43 @@ Economic governance MUST NOT encode:
 
 # VIII. Policy Supersession
 
-If a newer lawful transition conflicts with an existing pending transition:
+If a newer lawful economics policy version conflicts with an existing pending version:
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)